src/main/java/org/casbin/Client.java

```diff
@@ -21,6 +23,9 @@ public static String run(String... args) {
             if(Objects.equals(commandName, "-h") || Objects.equals(commandName, "--help")){
                 printHelpMessage();
                 return result;
+            } else if(Objects.equals(commandName, "-v") || Objects.equals(commandName, "--version")){
+                getVersion();
```

Comment:
Define `String getCliVersion()` and `String getCasbinVersion()`, return string, let Client.java do the print